src/tigerbeetle/cli.zig

```diff
@@ -496,11 +496,19 @@ pub fn parse_args(allocator: std.mem.Allocator, args_iterator: *std.process.ArgI
             const request_size_limit_min = 4096;
             const request_size_limit_max = constants.message_size_max;
             if (request_size_limit.bytes() > request_size_limit_max) {
-                flags.fatal("--limit-request: size {}{s} exceeds maximum: {}MiB", .{
-                    request_size_limit.value,
-                    request_size_limit.suffix(),
-                    @divExact(request_size_limit_max, 1024 * 1024),
-                });
+                if (comptime (request_size_limit_max >= 1024 * 1024)) {
+                    flags.fatal("--limit-request: size {}{s} exceeds maximum: {}MiB", .{
+                        request_size_limit.value,
+                        request_size_limit.suffix(),
+                        @divExact(request_size_limit_max, 1024 * 1024),
+                    });
+                } else {
+                    flags.fatal("--limit-request: size {}{s} exceeds maximum: {}KiB", .{
+                        request_size_limit.value,
+                        request_size_limit.suffix(),
+                        @divExact(request_size_limit_max, 1024),
+                    });
+                }
```

Comment:
Maybe this pattern warrants a formatter? Could be used elsewhere in this file for flexibility. Similar to `std.fmt.fmtIntSizeBin` except without the inexplicable floating-point conversion. (Using `@divExact`, no decimals).